kuma/static/js/interactive.js

```diff
@@ -41,12 +41,21 @@
         }
 
         if (event.data.label === 'Performance Events') {
+            // one of hitType: event
             mdn.analytics.trackEvent({
                 category: event.data.category,
                 action: event.data.action,
                 label: new Date().getTime() + '-' + randomString(),
                 value: event.data.value - performance.timing.fetchStart
             });
+
+            // one of hitType: timing
+            mdn.analytics.trackTiming({
+                category: event.data.category,
+                timingVar: event.data.action,
+                value: event.data.value - performance.timing.fetchStart,
+                label: new Date().getTime() + '-' + randomString()
```

Comment:
We don't have to be as funky here. Since it's a timing event, GA won't add all the values together so a unique label is not necessary. It's an optional string used to help in reporting. I think we can probably just ditch it altogether.

@jwhitlock what say you sir?

Reply:
As this field is also optional, I reckon do agree that we can simply leave it. Thanks for the feedback @tkadlec 

Will wait on @jw	 feedback as well before making the change though.

Reply:
Yes let's not set the label since is is optional. A similar change may be needed above to not send a label when omitted.